pulp_2to3_migration/app/plugin/docker/migrator.py

```diff
@@ -234,16 +234,16 @@ def relate_manifest_to_list(self, dc):
             for item in man_list:
                 if item.digest == digest:
                     break
-            platform = manifest['platform']
-            thru = ManifestListManifest(manifest_list=item, image_manifest=dc.content,
-                                        architecture=platform['architecture'],
-                                        os=platform['os'],
-                                        features=platform.get('features', ''),
-                                        variant=platform.get('variant', ''),
-                                        os_version=platform.get('os.version', ''),
-                                        os_features=platform.get('os.features', '')
-                                        )
-            mlm.append(thru)
+                platform = manifest['platform']
+                thru = ManifestListManifest(manifest_list=item, image_manifest=dc.content,
+                                            architecture=platform['architecture'],
+                                            os=platform['os'],
+                                            features=platform.get('features', ''),
+                                            variant=platform.get('variant', ''),
+                                            os_version=platform.get('os.version', ''),
+                                            os_features=platform.get('os.features', '')
+                                            )
+                mlm.append(thru)
```

Comment:
I think, this breaks the logic.
It should probably be:
```python
for item in man_list:
    if item.digest == digest:
        platform = manifest['platform']
        ...
        mlm.append(thru)
        break
```

Reply:
Thanks, I think you are right